examples/upgradeable-contract/README.md

```diff
@@ -0,0 +1,48 @@
+# Proxy Smart Contract
+
+The proxy smart contract delegates any call that does not match a
+selector of itself to another, specified contract.
+
+The instantiator of the proxy contract on a blockchain can change
+the code to which calls are forwarded.
+
+This allows building upgradeable contracts following [the proxy pattern](https://docs.openzeppelin.com/upgrades-plugins/1.x/proxies).
+
+In order to test it out you need to do the following:
+
+1. Build a contract containing some logic, e.g. our upgradeable flipper example:
+   ```
+   cargo +nightly contract build --manifest-path=examples/upgradeable-contract/upgradeable-flipper/Cargo.toml
+   ```
+   You will receive the respective `upgradeable_flipper.contract` bundle in the 
+   `examples/upgradeable-contract/upgradeable-flipper/target/ink/` folder.
+
+   In order to perform migrations and have proxy working for contracts with different 
+   [storage layouts](https://paritytech.github.io/ink-docs/datastructures/spread-storage-layout), 
+   we use the [`Upgradeable`](upgradeable-flipper/upgradeable.rs) type 
+   wrapper, which ensures that we write different fields of desired struct to different 
+   storage locations, while also tracking the initialization status (e.g., we uploaded 
+   the code on chain, but haven't called the constructor).
+
+3. Build the proxy contract:
```

Comment:
If you change all of these numbers to `1` Github is smart enough to render the the numbers in the correct order. Makes it easier when change are introduced (no need to update the entire list)